Dockerfile

```diff
@@ -1,7 +1,17 @@
-FROM ubuntu:16.04
+FROM ubuntu:18.04
 
 RUN apt-get update && \
-    apt-get -y install ruby ruby-dev git make build-essential libicu-dev zlib1g-dev cmake pkg-config
+    apt-get -y install ruby ruby-dev git make build-essential libicu-dev zlib1g-dev cmake pkg-config cmake libssl-dev
+
+# Set the locale
+# make the "en_US.UTF-8" locale so splunk will be utf-8 enabled by default
```

Comment:
perhaps we should drop the specific reference to splunk?

Reply:
That's what I get for copy and paste for a similar problem in another docker image I will remove the ref